planner/optimize.go

```diff
@@ -36,9 +36,14 @@ import (
 func Optimize(ctx context.Context, sctx sessionctx.Context, node ast.Node, is infoschema.InfoSchema) (plannercore.Plan, error) {
 	fp := plannercore.TryFastPlan(sctx, node)
 	if fp != nil {
+		if !IsPointGetWithoutDoubleRead(sctx, fp) {
+			sctx.PrepareTxnFuture(ctx)
```

Comment:
If `PrepareTxnFuture` is moved here, it can't run parallelly with `parse()`, and affect the latency of non-point-get requests.
Did you bench some cases to evaluate the benefit of the change?

Reply:
Already add the bench of point get with double read case in pr description.